static/js/containers/FinancialAidCalculator.js

```diff
@@ -0,0 +1,212 @@
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import { connect } from 'react-redux';
+import R from 'ramda';
+import Button from 'react-mdl/lib/Button';
+import TextField from 'material-ui/TextField';
+import Checkbox from 'react-mdl/lib/Checkbox';
+
+import {
+  updateCalculatorEdit,
+  clearCalculatorEdit,
+  addFinancialAid,
+  updateCalculatorValidation,
+} from '../actions/financial_aid';
+import { setCalculatorDialogVisibility } from '../actions/ui';
+import { createSimpleActionHelpers } from '../util/redux';
+import SelectField from '../components/inputs/SelectField';
+import { currencyOptions } from '../util/currency';
+import {
+  validateFinancialAid,
+  sanitizeNumberString
+} from '../util/validation';
+import type { ProgramEnrollment } from '../flow/enrollmentTypes';
+import type {
+  FinancialAidState,
+  FinancialAidValidation,
+} from '../reducers/financial_aid';
+
+const currencySelect = (update, current) => (
```

Comment:
We are not specifying a default currency based on the user's country of residence, right? Just making sure